search/src/main/scala/uk/ac/wellcome/platform/api/search/swagger/SwaggerDocs.scala

```diff
@@ -233,7 +232,9 @@ trait MultipleImagesSwagger {
             "cc-by-nc-nd",
             "cc-0",
             "pdm",
-            "copyright-not-cleared")),
+            "copyright-not-cleared"
```

Comment:
Let's not fix it here, as this diff is big enough, but this list is wrong – it's missing some licences, and other strings aren't actual licences. Tracked as https://github.com/wellcomecollection/platform/issues/5163